I will be using pandas to create tables, so importing the package will be helpful.

In [112]:
import pandas as pd

I will now scrape the website FBref for the ligue 1 table, where I will look at the teams that could potentially have some transfer targetted defenders.

In [115]:
Ligue1Table = pd.read_html('https://fbref.com/en/comps/13/Ligue-1-Stats', 
                  attrs = {"id":"results2024-2025131_overall"})[0]

In [116]:
display(Ligue1Table)

Rk          Squad  MP   W   D   L  GF  GA  GD  Pts  Pts/MP    xG   xGA  \
0    1      Paris S-G  34  26   6   2  92  35  57   84    2.47  88.9  31.2   
1    2      Marseille  34  20   5   9  74  47  27   65    1.91  63.8  45.2   
2    3         Monaco  34  18   7   9  63  41  22   61    1.79  72.8  34.8   
3    4           Nice  34  17   9   8  66  41  25   60    1.76  61.6  40.2   
4    5          Lille  34  17   9   8  52  36  16   60    1.76  54.7  42.5   
5    6           Lyon  34  17   6  11  65  46  19   57    1.68  60.3  45.7   
6    7     Strasbourg  34  16   9   9  56  44  12   57    1.68  49.3  56.1   
7    8           Lens  34  15   7  12  42  39   3   52    1.53  51.8  45.3   
8    9          Brest  34  15   5  14  52  59  -7   50    1.47  45.4  57.4   
9   10       Toulouse  34  11   9  14  44  43   1   42    1.24  49.1  36.8   
10  11        Auxerre  34  11   9  14  48  51  -3   42    1.24  41.0  58.6   
11  12         Rennes  34  13   2  19  51  50   1   41    1.21  48.6  46.0   
12  13         Nantes  34   8  12  14  39  52 -13   36    1.06  39.1  55.5   
13  14         Angers  34  10   6  18  32  53 -21   36    1.06  36.0  57.8   
14  15       Le Havre  34  10   4  20  40  71 -31   34    1.00  41.4  62.8   
15  16          Reims  34   8   9  17  33  47 -14   33    0.97  37.4  57.4   
16  17  Saint-Étienne  34   8   6  20  39  77 -38   30    0.88  36.4  74.3   
17  18    Montpellier  34   4   4  26  23  79 -56   16    0.47  33.5  63.3   

     xGD  xGD/90  Attendance                          Top Team Scorer  \
0   57.7    1.70       47413                     Ousmane Dembélé - 21   
1   18.6    0.55       63507                     Mason Greenwood - 21   
2   38.0    1.12       12164                        Mika Biereth - 13   
3   21.4    0.63       24299                      Evann Guessand - 12   
4   12.2    0.36       42417                      Jonathan David - 16   
5   14.6    0.43       51191                 Alexandre Lacazette - 15   
6   -6.9   -0.20       19378                      Emanuel Emegha - 14   
7    6.5    0.19       37971                      Neil El Aynaoui - 8   
8  -12.0   -0.35       14536                     Ludovic Ajorque - 13   
9   12.4    0.36       25593                    Zakaria Aboukhlal - 7   
10 -17.6   -0.52       16634  Gaëtan Perrin, Hamed Junior Traorè - 10   
11   2.6    0.08       27970                   Arnaud Kalimuendo - 17   
12 -16.4   -0.48       30317                       Matthis Abline - 9   
13 -21.8   -0.64       13142                       Esteban Lepaul - 9   
14 -21.4   -0.63       20218                     Abdoulaye Touré - 10   
15 -19.9   -0.59       15655                      Keito Nakamura - 11   
16 -37.9   -1.15       30220                       Lucas Stassin - 12   
17 -29.9   -0.91       13483                        Arnaud Nordin - 4   

              Goalkeeper                                        Notes  
0   Gianluigi Donnarumma         → Champions League via league finish  
1         Gerónimo Rulli         → Champions League via league finish  
2           Philipp Köhn         → Champions League via league finish  
3           Marcin Bułka         → Champions League via league finish  
4        Lucas Chevalier            → Europa League via league finish  
5            Lucas Perri            → Europa League via league finish  
6         Đorđe Petrović        → Conference League via league finish  
7            Brice Samba                                          NaN  
8            Marco Bizot                                          NaN  
9       Guillaume Restes                                          NaN  
10          Donovan Léon                                          NaN  
11           Brice Samba                                          NaN  
12         Anthony Lopes                                          NaN  
13          Yahia Fofana                                          NaN  
14         Arthur Desmas                             

I'll remove the attendance, top scorer, goalkeeper, and notes columns since they will not be used to look for defenders.

In [118]:
Ligue1Table = Ligue1Table.drop(['Attendance', 'Top Team Scorer', 'Goalkeeper', 'Notes'], axis=1)

In [120]:
display(Ligue1Table)

Rk          Squad  MP   W   D   L  GF  GA  GD  Pts  Pts/MP    xG   xGA  \
0    1      Paris S-G  34  26   6   2  92  35  57   84    2.47  88.9  31.2   
1    2      Marseille  34  20   5   9  74  47  27   65    1.91  63.8  45.2   
2    3         Monaco  34  18   7   9  63  41  22   61    1.79  72.8  34.8   
3    4           Nice  34  17   9   8  66  41  25   60    1.76  61.6  40.2   
4    5          Lille  34  17   9   8  52  36  16   60    1.76  54.7  42.5   
5    6           Lyon  34  17   6  11  65  46  19   57    1.68  60.3  45.7   
6    7     Strasbourg  34  16   9   9  56  44  12   57    1.68  49.3  56.1   
7    8           Lens  34  15   7  12  42  39   3   52    1.53  51.8  45.3   
8    9          Brest  34  15   5  14  52  59  -7   50    1.47  45.4  57.4   
9   10       Toulouse  34  11   9  14  44  43   1   42    1.24  49.1  36.8   
10  11        Auxerre  34  11   9  14  48  51  -3   42    1.24  41.0  58.6   
11  12         Rennes  34  13   2  19  51  50   1   41    1.21  48.6  46.0   
12  13         Nantes  34   8  12  14  39  52 -13   36    1.06  39.1  55.5   
13  14         Angers  34  10   6  18  32  53 -21   36    1.06  36.0  57.8   
14  15       Le Havre  34  10   4  20  40  71 -31   34    1.00  41.4  62.8   
15  16          Reims  34   8   9  17  33  47 -14   33    0.97  37.4  57.4   
16  17  Saint-Étienne  34   8   6  20  39  77 -38   30    0.88  36.4  74.3   
17  18    Montpellier  34   4   4  26  23  79 -56   16    0.47  33.5  63.3   

     xGD  xGD/90  
0   57.7    1.70  
1   18.6    0.55  
2   38.0    1.12  
3   21.4    0.63  
4   12.2    0.36  
5   14.6    0.43  
6   -6.9   -0.20  
7    6.5    0.19  
8  -12.0   -0.35  
9   12.4    0.36  
10 -17.6   -0.52  
11   2.6    0.08  
12 -16.4   -0.48  
13 -21.8   -0.64  
14 -21.4   -0.63  
15 -19.9   -0.59  
16 -37.9   -1.15  
17 -29.9   -0.91

Two good metrics to consider are GA (goals against) and xGA (expected goals against). I will give a rank of every team based on both categories mentioned, sum the rank and then order by this sum to find the best teams ranked by both metrics. 

In [123]:
Ligue1Table['GARk'] = Ligue1Table['GA'].rank()

In [124]:
Ligue1Table['xGARk'] = Ligue1Table['xGA'].rank()

In [126]:
Ligue1Table['OvrGARk'] = Ligue1Table['GARk'] + Ligue1Table['xGARk']

In [127]:
display(Ligue1Table.sort_values(by = ['OvrGARk']))

Rk          Squad  MP   W   D   L  GF  GA  GD  Pts  Pts/MP    xG   xGA  \
0    1      Paris S-G  34  26   6   2  92  35  57   84    2.47  88.9  31.2   
2    3         Monaco  34  18   7   9  63  41  22   61    1.79  72.8  34.8   
4    5          Lille  34  17   9   8  52  36  16   60    1.76  54.7  42.5   
3    4           Nice  34  17   9   8  66  41  25   60    1.76  61.6  40.2   
9   10       Toulouse  34  11   9  14  44  43   1   42    1.24  49.1  36.8   
7    8           Lens  34  15   7  12  42  39   3   52    1.53  51.8  45.3   
1    2      Marseille  34  20   5   9  74  47  27   65    1.91  63.8  45.2   
5    6           Lyon  34  17   6  11  65  46  19   57    1.68  60.3  45.7   
6    7     Strasbourg  34  16   9   9  56  44  12   57    1.68  49.3  56.1   
11  12         Rennes  34  13   2  19  51  50   1   41    1.21  48.6  46.0   
15  16          Reims  34   8   9  17  33  47 -14   33    0.97  37.4  57.4   
12  13         Nantes  34   8  12  14  39  52 -13   36    1.06  39.1  55.5   
10  11        Auxerre  34  11   9  14  48  51  -3   42    1.24  41.0  58.6   
8    9          Brest  34  15   5  14  52  59  -7   50    1.47  45.4  57.4   
13  14         Angers  34  10   6  18  32  53 -21   36    1.06  36.0  57.8   
14  15       Le Havre  34  10   4  20  40  71 -31   34    1.00  41.4  62.8   
16  17  Saint-Étienne  34   8   6  20  39  77 -38   30    0.88  36.4  74.3   
17  18    Montpellier  34   4   4  26  23  79 -56   16    0.47  33.5  63.3   

     xGD  xGD/90  GARk  xGARk  OvrGARk  
0   57.7    1.70   1.0    1.0      2.0  
2   38.0    1.12   4.5    2.0      6.5  
4   12.2    0.36   2.0    5.0      7.0  
3   21.4    0.63   4.5    4.0      8.5  
9   12.4    0.36   6.0    3.0      9.0  
7    6.5    0.19   3.0    7.0     10.0  
1   18.6    0.55   9.5    6.0     15.5  
5   14.6    0.43   8.0    8.0     16.0  
6   -6.9   -0.20   7.0   11.0     18.0  
11   2.6    0.08  11.0    9.0     20.0  
15 -19.9   -0.59   9.5   12.5     22.0  
12 -16.4   -0.48  13.0   10.0     23.0  
10 -17.6   -0.52  12.0   15.0     27.0  
8  -12.0   -0.35  15.0   12.5     27.5  
13 -21.8   -0.64  14.0   14.0     28.0  
14 -21.4   -0.63  16.0   16.0     32.0  
16 -37.9   -1.15  17.0   18.0     35.0  
17 -29.9   -0.91  18.0   17.0     35.0

As expected, PSG top the ranking for lowest on both metrics. Eyeing up these defenders for transfers are harder, since they play for a more profoud team, and are less likely to move, especially after the champions league win. Monaco, Lille and Nice all have a reasonable low rating, and being the teams placed 3-5 for points, this would be expected. One team that has ranked significantly lower than their position is Marseille, having conceded the 9th most goals, despite finishing 2nd. Another team that ranked noticably different is Toulouse, however their defensive rank is much higher than their position in the table.

I wanted to take a look at Toulouse's results, to see how their record faired over the league, specifically the GA record.

In [131]:
TResults = pd.read_html('https://fbref.com/en/squads/3f8c4b5f/Toulouse-Stats', 
                  attrs = {"id":"matchlogs_for"})[0]

In [132]:
TResults = TResults[TResults['Comp'] == 'Ligue 1']

Taking a closer look, they only conceded 3 or more goals 4 times in the league, shown below.

In [136]:
sum(TResults['GA'].astype(int) >= 3)

4

For a mid table team, this seemed impressive. I wanted to check if this achievement was mainly down to the goalkeepr. Below are the stats of their main goalkeeper, Guillaume Restes, having started 29 out of the 34 games.

In [140]:
GR = pd.read_html('https://fbref.com/en/players/6b91b999/Guillaume-Restes', 
                  attrs = {"id":"scout_summary_GK"})[0]

In [141]:
display(GR)

Statistic Per 90  Percentile
0                          PSxG-GA  -0.20         9.0
1                    Goals Against   1.27        58.0
2                  Save Percentage  60.7%         5.0
3                         PSxG/SoT   0.29        35.0
4            Save% (Penalty Kicks)  50.0%        86.0
5           Clean Sheet Percentage  29.6%        69.0
6                              NaN    NaN         NaN
7                          Touches  30.68         7.0
8                         Launch %  30.8%        45.0
9                       Goal Kicks   4.78        34.0
10       Avg. Length of Goal Kicks   42.8        44.0
11                             NaN    NaN         NaN
12               Crosses Stopped %   1.4%         1.0
13  Def. Actions Outside Pen. Area   1.20        63.0
14   Avg. Distance of Def. Actions   16.9        88.0

Looking at the above table, Restes achieved a save percentage of 60.7%, ranking in the bottom 5%. Ultimately, their defensive record did not seem down to their goalkeeper, so I'll have a further look at their defenders and see who performed best. Below is the table of all Toulouse defenders.

In [143]:
TDF = pd.read_html('https://fbref.com/en/squads/3f8c4b5f/Toulouse-Stats', 
                  attrs = {"id":"stats_standard_13"})[0]

In [144]:
TDF.columns = TDF.columns.get_level_values(1)

In [145]:
TDF = TDF[TDF['Pos'] == 'DF']

In [146]:
display(TDF)

Player   Nation Pos   Age  MP  Starts     Min   90s  Gls  Ast  \
2   Charlie Cresswell  eng ENG  DF  21.0  31      29  2592.0  28.8  3.0  1.0   
4       Mark McKenzie   us USA  DF  25.0  30      29  2496.0  27.7  1.0  1.0   
7      Djibril Sidibé   fr FRA  DF  32.0  28      24  2101.0  23.3  0.0  3.0   
8       Gabriel Suazo   cl CHI  DF  26.0  30      23  2163.0  24.0  0.0  6.0   
15  Rasmus Nicolaisen   dk DEN  DF  27.0   7       7   623.0   6.9  0.0  1.0   
16         Ümit Akdag   ro ROU  DF  20.0   9       6   500.0   5.6  0.0  0.0   
18      Rafik Messali   fr FRA  DF  21.0  13       5   456.0   5.1  0.0  1.0   
23        Logan Costa   cv CPV  DF  23.0   1       1    90.0   1.0  0.0  0.0   
25       Edhy Zuliani   fr FRA  DF  19.0   1       0     9.0   0.1  0.0  0.0   
30    Dayann Methalie   fr FRA  DF  18.0   0       0     NaN   NaN  NaN  NaN   

    ...   Ast   G+A  G-PK  G+A-PK    xG   xAG  xG+xAG  npxG  npxG+xAG  Matches  
2   ...  0.03  0.14  0.10    0.14  0.11  0.06    0.17  0.11      0.17  Matches  
4   ...  0.04  0.07  0.04    0.07  0.05  0.02    0.07  0.05      0.07  Matches  
7   ...  0.13  0.13  0.00    0.13  0.02  0.13    0.14  0.02      0.14  Matches  
8   ...  0.25  0.25  0.00    0.25  0.03  0.16    0.19  0.03      0.19  Matches  
15  ...  0.14  0.14  0.00    0.14  0.06  0.03    0.10  0.06      0.10  Matches  
16  ...  0.00  0.00  0.00    0.00  0.01  0.01    0.03  0.01      0.03  Matches  
18  ...  0.20  0.20  0.00    0.20  0.05  0.20    0.25  0.05      0.25  Matches  
23  ...  0.00  0.00  0.00    0.00  0.00  0.50    0.50  0.00      0.50  Matches  
25  ...  0.00  0.00  0.00    0.00  0.00  0.00    0.00  0.00      0.00  Matches  
30  ...   NaN   NaN   NaN     NaN   NaN   NaN     NaN   NaN       NaN  Matches  

[10 rows x 34 columns]

We can see that the players that were crucial to Toulouse's defence was Charlie Cresswell, Mark McKenzie, Djibril Sidibé, and Gabriel Suazo. I wanted to take a closer look at Charlie Cresswell, a young English player who just finished his first season in Ligue 1, after a move from Leeds United.

In [151]:
CC = pd.read_html('https://fbref.com/en/players/4654e5f5/Charlie-Cresswell', 
                  attrs = {"id":"scout_summary_CB"})[0]

In [152]:
CC = CC.dropna()

In [153]:
display(CC)

Statistic Per 90  Percentile
0          Non-Penalty Goals   0.10        91.0
1       npxG: Non-Penalty xG   0.11        99.0
2                Shots Total   1.11        98.0
3                    Assists   0.03        66.0
4   xAG: Exp. Assisted Goals   0.06        91.0
5                 npxG + xAG   0.17        99.0
6      Shot-Creating Actions   0.83        50.0
8           Passes Attempted  53.85        44.0
9          Pass Completion %  83.0%        18.0
10        Progressive Passes   2.40        25.0
11       Progressive Carries   0.28        20.0
12       Successful Take-Ons   0.10        28.0
13         Touches (Att Pen)   1.94        98.0
14    Progressive Passes Rec   0.45        75.0
16                   Tackles   1.22        27.0
17             Interceptions   0.97        39.0
18                    Blocks   1.53        79.0
19                Clearances   6.63        89.0
20               Aerials Won   3.40        95.0

Overall, for a 21 year old in his first season in a different league, I'd say his stats are something to be interested in. First, his offensive stats.

In [157]:
display(CC.iloc[0:7])

Statistic Per 90  Percentile
0         Non-Penalty Goals   0.10        91.0
1      npxG: Non-Penalty xG   0.11        99.0
2               Shots Total   1.11        98.0
3                   Assists   0.03        66.0
4  xAG: Exp. Assisted Goals   0.06        91.0
5                npxG + xAG   0.17        99.0
6     Shot-Creating Actions   0.83        50.0

Scoring 3 goals and being in the top 10% for non-penalty goals, he clearly offers threat in the opponents box. Additionally, being in the top 1% for non-pentaly xG clearly shows despite his position, he can help the team when he needs to. Now, we'll take a look at his passing and controlling stats.

In [159]:
display(CC.iloc[7:14])

Statistic Per 90  Percentile
8         Passes Attempted  53.85        44.0
9        Pass Completion %  83.0%        18.0
10      Progressive Passes   2.40        25.0
11     Progressive Carries   0.28        20.0
12     Successful Take-Ons   0.10        28.0
13       Touches (Att Pen)   1.94        98.0
14  Progressive Passes Rec   0.45        75.0

His possession based stats do not bode well, having just below average attempted passes and wrose pass completion and progressive passes. However, his touches inside the oppositions attacking penalty area are one of the highest among the top 5 leagues, as well as being in the top 25% for progressive passess recorded. He clearly enjoys playing a more advanced role. Last and most importantly, we look at his defensive stats.

In [165]:
display(CC.iloc[14:19])

Statistic Per 90  Percentile
16        Tackles   1.22        27.0
17  Interceptions   0.97        39.0
18         Blocks   1.53        79.0
19     Clearances   6.63        89.0
20    Aerials Won   3.40        95.0

His tackles and interceptions are among the lower percentile for the top 5 leagues, achieving 1.22 tackles per 90. However, his 1.53 blocks per 90 ranks high, as well as his clearences and aerials won, despite only being 6ft.

I took a closer look at his goals, scoring 3 in the league and 1 in the french cup. 3 headers, 1 tap in from a rebound. He offers great threat in the box, however as a defender, better tackling and interceptions may be more valuable.

At such a young age, I think there's a lot of 

Let's take a look at one of Cresswell's other defensive teammates, Mark McKenzie.

In [170]:
MM = pd.read_html('https://fbref.com/en/players/a5f5d094/scout/365_m1/Mark-McKenzie-Scouting-Report', 
                  attrs = {"id":"scout_full_CB"})[0]

In [171]:
MM.columns = MM.columns.get_level_values(1)
MM = MM.dropna()
MM = MM[MM['Statistic'] != 'Statistic']
MM = MM[MM['Per 90'] != 'Shooting']
MM = MM[MM['Per 90'] != 'Passing']
MM = MM[MM['Per 90'] != 'Pass Types']
MM = MM[MM['Per 90'] != 'Goal and Shot Creation']
MM = MM[MM['Per 90'] != 'Defense']
MM = MM[MM['Per 90'] != 'Possession']
MM = MM[MM['Per 90'] != 'Miscellaneous Stats']

In [174]:
display(MM.iloc[15:30])

Statistic Per 90 Percentile
18                     Goals   0.04         52
19               Shots Total   0.29         19
20           Shots on Target   0.07         25
21         Shots on Target %  25.0%         41
22                Goals/Shot   0.13         74
23      Goals/Shot on Target   0.50         77
24     Average Shot Distance  11.20         65
25     Shots from Free Kicks   0.00         44
26        Penalty Kicks Made   0.00         49
27   Penalty Kicks Attempted   0.00         49
29        xG: Expected Goals   0.05         57
30      npxG: Non-Penalty xG   0.05         58
31                 npxG/Shot   0.17         94
32                Goals - xG  -0.01         43
33  Non-Penalty Goals - npxG  -0.01         42

His offensive stats aren't anything out of the ordinary, but as a defender this is not the key part. Let's take a look at his defensive stats.

In [178]:
display(MM.iloc[81:97])

Statistic Per 90 Percentile
102                 Tackles   1.88         79
103             Tackles Won   1.12         77
104       Tackles (Def 3rd)   1.12         77
105       Tackles (Mid 3rd)   0.72         86
106       Tackles (Att 3rd)   0.04         28
108       Dribblers Tackled   1.19         90
109     Dribbles Challenged   1.62         82
110  % of Dribblers Tackled  73.3%         83
111         Challenges Lost   0.43         47
113                  Blocks   1.12         38
114           Shots Blocked   0.65         40
115          Passes Blocked   0.47         45
117           Interceptions   0.79         19
118                 Tkl+Int   2.67         55
119              Clearances   5.08         58
120                  Errors   0.04         75

McKenzie has impressive defensive stats, ranking in the top 25% for all tackles (besides tackles in the attacking third). Ranking high for dribblers tackled and high for errors leading to a shot. Overall, McKenzie has a record scouts should be monitoring, especially at such a young age.

Finally, I'd like to take a look at Gabriel Suazo, who plays as a fullback, but likes to play in more advanced areas like midfield, specifically on the left.

In [183]:
GS = pd.read_html('https://fbref.com/en/players/39b326ed/scout/365_m1/Gabriel-Suazo-Scouting-Report', 
                  attrs = {"id":"scout_full_FB"})[0]

In [184]:
GS.columns = GS.columns.get_level_values(1)
GS = GS.dropna()
GS = GS[GS['Statistic'] != 'Statistic']
GS = GS[GS['Per 90'] != 'Shooting']
GS = GS[GS['Per 90'] != 'Passing']
GS = GS[GS['Per 90'] != 'Pass Types']
GS = GS[GS['Per 90'] != 'Goal and Shot Creation']
GS = GS[GS['Per 90'] != 'Defense']
GS = GS[GS['Per 90'] != 'Possession']
GS = GS[GS['Per 90'] != 'Miscellaneous Stats']

In [185]:
display(GS.iloc[30:52])

Statistic  Per 90 Percentile
36              Passes Completed   41.36         55
37              Passes Attempted   51.93         54
38             Pass Completion %   79.6%         53
39        Total Passing Distance  659.83         53
40  Progressive Passing Distance  271.04         79
42      Passes Completed (Short)   22.26         61
43      Passes Attempted (Short)   25.05         63
44     Pass Completion % (Short)   88.9%         42
46     Passes Completed (Medium)   15.52         47
47     Passes Attempted (Medium)   19.14         47
48    Pass Completion % (Medium)   81.1%         49
50       Passes Completed (Long)    2.37         43
51       Passes Attempted (Long)    4.95         47
52      Pass Completion % (Long)   47.9%         43
53                       Assists    0.25         94
54      xAG: Exp. Assisted Goals    0.16         82
55          xA: Expected Assists    0.20         92
56                    Key Passes    1.66         92
57       Passes into Final Third    3.12         60
58      Passes into Penalty Area    1.58         91
59     Crosses into Penalty Area    0.50         62
60            Progressive Passes    4.54         75

Suazo shows an impressive overall passing record, especially in those bottom statistics. One of the best for assists in his position, achieving 6 assists in the season. He also ranked in the top 10% for key passes, an important statistic for a midfielder. Ultimately, Suazo has a very impressive passing record. Finally, let's have look at his defensive record.

In [187]:
display(GS.iloc[81:97])

Statistic Per 90 Percentile
102                 Tackles   2.45         76
103             Tackles Won   1.62         80
104       Tackles (Def 3rd)   1.04         45
105       Tackles (Mid 3rd)   0.92         81
106       Tackles (Att 3rd)   0.50         96
108       Dribblers Tackled   1.25         65
109     Dribbles Challenged   3.00         95
110  % of Dribblers Tackled  41.7%          4
111         Challenges Lost   1.75          1
113                  Blocks   1.12         54
114           Shots Blocked   0.12         10
115          Passes Blocked   1.00         75
117           Interceptions   1.50         89
118                 Tkl+Int   3.95         84
119              Clearances   1.66         12
120                  Errors   0.04         56

Some things that stand out are his challenges lost, placing in the bottom 1%, and a low ranking for his % of dribblers tackled. Besides this, he has some good ratings, with high interceptions and very high tackles in the attacking third.

In [189]:
display(GS[GS['Statistic'] == 'Goal-Creating Actions'])
display(GS[GS['Statistic'] == 'Shot-Creating Actions'])

Statistic Per 90 Percentile
92  Goal-Creating Actions   0.46         94

Statistic Per 90 Percentile
83  Shot-Creating Actions   3.41         94

I'd also like to point out his shot and goal creating actions, ranking in the top 6% for both, he provides threat directly or indirectly for his team.

To conclude, I think Toulouse have some defenders that teams should look out for, McKenzie providing solid defensive work, Cresswell delivering some attacking threat in the box, and Suazo creating offensive chances as well as providing defense. Given Cresswell's age, I think it's best for him to build a credible record staying at Toulouse, and could look for a transfer back to England. With Suazo's strong progressive stats and attacking threat, I think teams should really consider picking him out. He's not afraid to drive at defenders, and with short bursts of speed, he offers great chances down that left side, as well as putting his defensive duties in.